backend/amd64/emit.mlp

```diff
@@ -65,12 +65,18 @@ let cfi_adjust_cfa_offset n =
 let emit_debug_info ?discriminator dbg =
   emit_debug_info_gen ?discriminator dbg D.file D.loc
 
-let emit_debug_info_linear i =
-  match i.fdo with
-  | None -> emit_debug_info i.dbg
+let emit_debug_info_helper ~out_of_line fdo dbg =
+  match (fdo : Fdo_info.t) with
+  | None -> emit_debug_info dbg
   | Some { discriminator; dbg } ->
+    let discriminator =
+      if out_of_line then (-discriminator) else  discriminator
```

Comment:
```suggestion
      if out_of_line then (-discriminator) else discriminator
```
(Should we enable `ocamlformat` on that file,
or is it disabled to keep merges manageable?)

Reply:
yes, I want to enable on all of `backend` folder  recursively.